code/test_resid_normality.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np 
```

Comment:
you can remove this file too because lynn added this file in our codebase. Thanks!